pkg/workflows/sdk/builder.go

```diff
@@ -238,3 +238,35 @@ func AnyMap[M ~map[string]any](inputs CapMap) CapDefinition[M] {
 
 	return components
 }
+
+// Alt 1:
+func XSecrets() string {
+	return "$(ENV.secrets)"
+}
+
+func XSecretNamed(name string) string {
+	return fmt.Sprintf("$(ENV.secrets.%s)", name)
+}
```

Comment:
I like this one, but I'm opened to either approach.  

Maybe just secret, no need for Named.  It's common pattern to have foos() and foo(name string).  I wonder if they should return a special type

type Seceret string

then you can implement CapDefinition[String] with so it can be passed as input to a capability, and it can also easily be cast to a string.  Thoughts?

Reply:
Please take another look and let me know what you think :) Note: atm secrets (and env var interpolation in general) only works on Config; but we could easily change this if necessary.